### PR TITLE
Added demos for list view grid component

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "@skyux/grids": "^4.0.0",
     "@skyux/i18n": "^4.0.0",
     "@skyux/indicators": "^4.0.0",
-    "@skyux/list-builder": "^4.5.0",
+    "@skyux/list-builder": "^4.0.2",
```
Comment (Blackbaud-AlexKingman) [inline package.json:29]: ☝️ Fixing an error from a prior commit. Unrelated to this PR.

File: src/app/public/plugin-resources/code-examples/list-view-grid/templated-column/list-view-grid-demo.component.scss [hunk 2]
```diff
@@ -0,0 +1,6 @@
+@import "~@skyux/theme/scss/mixins";
+@import "~@skyux/theme/scss/_compat/mixins";
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/plugin-resources/code-examples/list-view-grid/templated-column/list-view-grid-demo.component.scss]: We probably don't want to expose the `_compat` styles to our consumers.

Comment (Blackbaud-AlexKingman) [reply src/app/public/plugin-resources/code-examples/list-view-grid/templated-column/list-view-grid-demo.component.scss]: Removed.

File: src/app/public/modules/list-view-grid/list-view-grid.component.spec.ts [hunk 3]
```diff
@@ -537,110 +537,97 @@ describe('List View Grid Component', () => {
         fixture.detectChanges();
         tick();
         fixture.detectChanges();
-        expect(fixture.nativeElement.querySelector('.sky-grid-row-delete-heading')).toBeNull();
+        expect(document.querySelector('#row-delete-ref-1')).toBeNull();
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/list-view-grid/list-view-grid.component.spec.ts:540]: All these changes to the spec file were to account for a change from this PR: https://github.com/blackbaud/skyux-grids/pull/157/files.

They are unrelated to this PR.

**Review by Blackbaud-SteveBrush — DISMISSED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-list-builder-view-grids?_sp.spa=pr-demo](https://developer.blackbaud.com/skyux-list-builder-view-grids?_sp.spa=pr-demo)

**Review by Blackbaud-DeniseCisneros — DISMISSED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-list-builder-view-grids?_sp.spa=pr-demo](https://developer.blackbaud.com/skyux-list-builder-view-grids?_sp.spa=pr-demo)

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**